optuna_dashboard/preferential/_study.py

```diff
@@ -182,6 +182,57 @@ def add_trials(self, trials: Iterable[FrozenTrial]) -> None:
         """
         self._study.add_trials(trials)
 
+    def enqueue_trial(
+        self,
+        params: dict[str, Any],
+        user_attrs: dict[str, Any] | None = None,
+        skip_if_exists: bool = False,
+    ) -> None:
+        """Enqueue a trial with given parameter values.
+
+        You can fix the next sampling parameters which will be evaluated in your
+        objective function.
+
+        Example:
+
+            .. testcode::
+
+                import optuna
+
+
+                def objective(trial):
+                    x = trial.suggest_float("x", 0, 10)
+                    return x**2
+
+
+                study = optuna.create_study()
+                study.enqueue_trial({"x": 5})
+                study.enqueue_trial({"x": 0}, user_attrs={"memo": "optimal"})
+                study.optimize(objective, n_trials=2)
+
+                assert study.trials[0].params == {"x": 5}
+                assert study.trials[1].params == {"x": 0}
+                assert study.trials[1].user_attrs == {"memo": "optimal"}
```

Comment:
nit: How about removing the example code and placing a link to the Optuna documentation instead, just like with the other methods?

```suggestion
        .. seealso::

            See `Study.enqueue_trials`_ for details.

            .. _Study.get_trials: https://optuna.readthedocs.io/en/stable/reference/\
            generated/optuna.study.Study.html#optuna.study.Study.enqueue_trials
```